samples/javascript_nodejs/26.transcript-logger-bot/README.md

```diff
@@ -8,15 +8,24 @@ In this example, the bot creates a new folder in the project directory called `l
 
  This bot example uses [restify](https://www.npmjs.com/package/restify).
 
-# To run the bot
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+- In a terminal, navigate to `samples/javascript_nodejs/26.bot-transcript-logging`
+    ```bash
+    cd samples/javascript_nodejs/26.bot-transcript-logging
+    ```
+- [Optional] Update the .env file under `samples/javascript_nodejs/26.bot-transcript-logging` with your botFileSecret For Azure Bot Service bots, you can find the botFileSecret under application settings.
```

Comment:
Since `botFileSecret` is a variable in the `.env` file. It should be wrapped in `code style` backticks.
```suggestion
- [Optional] Update the .env file under `samples/javascript_nodejs/26.bot-transcript-logging` with your `botFileSecret` For Azure Bot Service bots, you can find the `botFileSecret` under application settings.
```